pkg/cri/config/config_unix.go

```diff
@@ -100,7 +100,7 @@ func DefaultConfig() PluginConfig {
 		},
 		EnableCDI:                false,
 		CDISpecDirs:              []string{"/etc/cdi", "/var/run/cdi"},
-		ImagePullProgressTimeout: time.Minute.String(),
+		ImagePullProgressTimeout: (5 * time.Minute).String(),
```

Comment:
Just a quick blurb, per my comment on the other PR, it may be good to document where this timeout came from (why we chose this value); which could be a `const` somewhere that has the documentation on it.